form/src/main/java/org/dhis2/form/ui/provider/inputfield/ImageInputProvider.kt

```diff
@@ -44,7 +46,10 @@ internal fun ProvideInputImage(
         load = {
             painter
         },
-        onDownloadButtonClick = { fieldUiModel.invokeUiEvent(UiEventType.OPEN_FILE) },
+        onDownloadButtonClick = {
+            fieldUiModel.setFocus()
```

Comment:
is this needed?, if not is not getting the focus?

Reply:
this is due to this                
 viewModel.getFocusedItemUid()?.let {
                    viewModel.submitIntent(FormIntent.OnAddImageFinished(it))
                } sometimes not returning anything 
and causing an exception I wanted to ask you about this in the daily, I will remove it in a second and do some testing again